types/sitemap-generator/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for sitemap-generator 8.5
+// Project: https://github.com/lgraubner/sitemap-generator
+// Definitions by: grgr-dkrk <https://github.com/grgr-dkrk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 2.8
```

Comment:
```suggestion
```
Is there any specific reason for backward copatiblity? The most recent supported is 3.2 and newer (by default)